onnxruntime/core/providers/dml/DmlExecutionProvider/src/DmlGraphFusionHelper.h

```diff
@@ -39,20 +39,6 @@ namespace DmlGraphFusionHelper
         ID3D12Resource** resource,
         uint64_t* allocId);
 
-    void ProcessInputData(
```

Comment:
This function is only used by `DmlGraphFusionHelper.cpp`, so no need to have it in the header